managing_providers/_topics/embedded_workflows.md

```diff
@@ -187,11 +187,26 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
       
       You are recommended to use a docker.io [access token](https://docs.docker.com/security/for-developers/access-tokens/) so that the token does not expire.
 
-   * For OCP provide an image pull secret to a podified Kubernetes container, and then add it to a service  account
+   * Provide an image pull secret to a podified Kubernetes container, and then add it to a service account
```

Comment:
Is this section needed now that we have `#### Running an Embedded Workflow on the Openshift Container Platform (OCP) ` below?